Automattic/wp-calypso
# Jetpack Connect: Less scary illustration for NoDirectAccessError and JetpackSsoForm

I was working on #24214 and while testing I felt that the current illustration we're using in that case seemed a little too scary and maybe a little bit unfriendly. Just to clarify, this happens when the users directly visit URLs that they're not supposed to visit:

* https://wordpress.com/jetpack/connect/authorize
* https://wordpress.com/jetpack/sso

Perhaps we could use a more suitable one? I think we have a quick winner here, but I could use some design feedback on whether the suggestion makes sense. 

**Authorize step error - before**
![](https://cldup.com/6CiQ_HDq6x.png)

**Authorize step error - after**
![](https://cldup.com/pJwssjc7TZ.png)

**SSO error - before**
![](https://cldup.com/UCZseIy7vR.png)

**SSO error - after**
![](https://cldup.com/l8k-PT7MrY.png)

To test:
* Visit https://calypso.live/jetpack/connect/authorize?branch=update/jpc-no-direct-access-illustration
* Verify it looks as shown on the **Authorize step error - after** screenshot.
* Visit https://calypso.live/jetpack/sso?branch=update/jpc-no-direct-access-illustration
* Verify it looks as shown on the **SSO error - after** screenshot.
